_benchmark/user-guide/contributing-workloads.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: Contributing custom workloads
+nav_order: 11
+parent: User guide
+---
+
+# Contributing custom workloads
+
+After [creating a custom workload]({{site.url}}{{site.baseurl}}/benchmark/creating-custom-workloads/) you can contribute your workload for other OpenSearch Benchmark users to use. If you plan on contibuting a workload, use the following guidelines.
+
+## README and license
+
+Each workload must include a detailed `README.MD` file, which should include the following:  
+
+- The purpose of the workload and what users should use the workload. When creating a description, think about what specific use case your workload address that other workloads included in OpenSearch Benchmark do not.
+- An overview of the dataset used by the workload. Make sure that any data included in the dataset does not contain any proprietary data or Personal Identifiable Information (PII). 
+- An example document from the data set that gives users an idea of how the data looks like.
+- The parameters supported by the workload.
+- A list of default test procedures included in the workload and other test procedures the workload can run.
```

Comment:
- A list of the test procedures included and an identification of which is the default test procedure.